src/components/Generic/Dialogs/AddOrUpdatePrinterDialog.vue

```diff
@@ -56,22 +42,33 @@
                         required
                       ></v-checkbox>
                     </validation-provider>
-
+                  </v-row>
+                  <div style="height: 40px"></div>
```

Comment:
Empty div

Reply:
I added these as a padding/empty space between the elements to so elements dont intersect. Not sure what is the better replacement. V-spacer?